packages/vike-react-styled-components/README.md

```diff
@@ -0,0 +1,113 @@
+# `vike-react-styled-components`
+
+[Installation](#installation)  
+[Settings](#settings)  
+[Version history](https://github.com/vikejs/vike-react/blob/main/packages/vike-react-styled-components/CHANGELOG.md)  
+[See Also](#see-also)  
+
+<br/>
+
+Integrates [styled-components](https://styled-components.com) to your [`vike-react`](https://vike.dev/vike-react) app.
+
+## Installation
+
+1. `npm install vike-react-styled-components styled-components`
+   
+2. Extend `+config.js`:
+   ```js
+   // pages/+config.js
+
+   import vikeReact from "vike-react/config"
+   import vikeReactStyledComponents from "vike-react-styled-components/config"
+
+   export default {
+     // ...
+     extends: [vikeReact, vikeReactStyledComponents]
+   }
+   ```
+3. You can now use styled-components at any of your components.
+   ```jsx
+   import { styled } from "styled-components";
+   const Title = styled.h1`
+      font-size: 1.5em;
+      text-align: center;
+      color: #BF4F74;
+    `;
+
+   const Wrapper = styled.section`
+      padding: 4em;
+      background: papayawhip;
+    `;
+
+   function SomeComponent() {
+     return (
+       <Wrapper>
+        <Title>Hello World!</Title>
+      </Wrapper>
+     )
+   }
+   ```
+
+> [!NOTE]
+> The `vike-react-styled-components` extension requires [`vike-react`](https://vike.dev/vike-react).
+
+
+In order to reliably perform server side rendering and avoids checksum mismatches due to different class generation on the client and on the server, you'll need to use [babel-plugin-styled-components](https://styled-components.com/docs/tooling#babel-plugin):
```

Comment:
Is the babel plugin optional or is it required?

Reply:
Not entirely, only in development. I forgot to mention that

Reply:
Still, all-in-all, is the babel plugin optional or is it required? Is other words: does the app work if I don't install the babel plugin?

Reply:
Yes, it still works without the Babel plugin. 

The only issue is that if we change the style during development, the class may mismatch between the server and client.

Reply:
> the class may mismatch between the server and client.

That sounds bad? How about we add the babel instruction to the installation steps?

If you feel like it's useful, we can add a note that it's optional. Although the mismatch thing sounds bad enough that I'd be inclined to not even add the note.

Reply:
In other words: it sound impractical to use it without the plugin, so I wouldn't mention that it's optional.

Reply:
> How about we add the babel instruction to the installation steps?

Sure, we can do that. I thought about that too before